### ISSUE TITLE
How are packages added to this repository?

### ISSUE THREAD

**Issue by branb** [magento-hackathon/composer-repository#451]
Even though I don't know exactly how, I understand that the connect20-Packages are crawled regularly. What about the other packages? Can I, for example, add packages? If so, how?

**Comment by Schrank** [comment]
Yes, send a PR and add your VCS link - many examples in the closed PRs.
